src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,37 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ id, liked, sender, body, timeStamp, onLikeChange, isLocal, style }) => {
+  const handleLikeClick = () => {
+    onLikeChange(id, !liked);
```

Comment:
Clever to handle the switching of the value here so you can use a shorter ternary expression in App.jsx